examples/cloudml-sklearn-pipeline/.pylintrc

```diff
@@ -0,0 +1,334 @@
+[MASTER]
```

Comment:
I am not sure we should submit dot files, in particular pylint - the contributor guidelines already refer to the Python Style Guide

Reply:
Since this will be eventually used as standalone template or tool, i think it is still make sense to have it.